org.syphon.Syphon.json

```diff
@@ -0,0 +1,97 @@
+{
+    "app-id": "org.syphon.Syphon",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "41",
+    "sdk": "org.gnome.Sdk",
+    "command": "syphon",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=all",
+        "--filesystem=xdg-download",
+        "--talk-name=org.freedesktop.Notifications",
+        "--own-name=org.syphon.Syphon"
+    ],
+    "modules": [
+        {
+            "name": "libjsoncpp",
+            "buildsystem": "cmake",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=release",
+                "-DBUILD_SHARED_LIBS=ON"
+            ],
```

Comment:
```suggestion
            ],
            "builddir": true,
```

This is what I would try to solve the build problem.

BTW I'm still surprised you need to build these dependencies since you unpack a binary distribution.

Reply:
Doesn't work as this is an array not a map. Should this be somewhere else?

Reply:
This is not what the "suggestion" says.

Reply:
Aha, indeed it does not. Got confused because GitHub highlighted too much in red.

![image](https://user-images.githubusercontent.com/83190128/146687923-17adfb4e-04c2-44c0-878c-91bf6ed27722.png)

In local testing, that fixes it, thanks! :D

Reply:
>BTW I'm still surprised you need to build these dependencies since you unpack a binary distribution.

For some reason, flutter seems to include some libs and not others. On that topic also, using version 1.7.5 works but 1.9.5 produces `syphon: error while loading shared libraries: libjsoncpp.so.1: cannot open shared object file: No such file or directory`, even if I add `ln -s -r /app/lib/libjsoncpp.so /app/lib/libjsoncpp.so.1`. Any idea what could be going wrong?

Reply:
Turns out, it's just incompatible